### PR TITLE
add lesson04 Task01-10

### PR DESCRIPTION
**Pull request by haretski-aliaksei**
## Pull request state

Actual status for PR:
- [x] implemented
- [x] reviewed
- [ ] fixed
- [ ] approved

### CODE REVIEW

File: src/main/java/com/rakovets/course/java/core/practice/looping_statements/Task09.java [hunk 1]
```diff
@@ -34,8 +34,37 @@ public static void main(String[] args) {
...
+
+        for (int i = numberOfCharactersInNumber - COUNTER; i >= 0; i--) {
+            currentCharacter = (long) (amount / Math.pow(10, i));
+            amountConvertedToAccountingFormat += currentCharacter;
+            amount -= currentCharacter * Math.pow(10, i);
+
+            if (i % SEPARATE_INDICATOR == 0 && i != 0) {
+                amountConvertedToAccountingFormat += WHITE_SPACE;
+            }
+        }
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/looping_statements/Task09.java:67]: hard

File: src/main/java/com/rakovets/course/java/core/practice/looping_statements/Task10.java [hunk 2]
```diff
@@ -38,8 +38,26 @@ public static void main(String[] args) {
...
+                    || i == 5
+                    || i == 7
+                    || i == 11
+                    || i == 13
+                    || i == 17
+                    || i == 19
+                    || i == 23) {
+                primeNumber++;
+            }
+        }
```
Comment (rakovets) [inline src/main/java/com/rakovets/course/java/core/practice/looping_statements/Task10.java:60]: wrong solution

**Review by rakovets — CHANGES_REQUESTED** [review]
Need to add correct solution

**Comment by haretski-aliaksei** [comment]
Fixed PR: https://github.com/rakovets/course-java-core/pull/975

**Pull request closed without merging**